src/Components/Navigation.jsx

```diff
@@ -50,6 +51,18 @@ const useStyles = makeStyles(theme => ({
             transform: "translateX(0)",
         },
     },
+    navCalc: {
+        height: "40px",
```

Comment:
Should we use `rem` or `em` instead of `px` here? Then we can try to account for different size and resolutions of screens